libparsec/crates/platform_mountpoint/src/error.rs

```diff
@@ -0,0 +1,19 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS
+
+use thiserror::Error;
+
+#[derive(Debug, Error)]
+pub(crate) enum MountpointError {
```

Comment:
Should the variants have more context ? I feel likes it's missing the path that generate the error :thinking:

Reply:
`Windows` doesn't handle any path

Reply:
How that possible ? to open a file I need a path

Reply:
It uses `GetLastError` function

Reply:
It is called automatically for the user

Reply:
I'm not sure to understand why `GetLastError` can't be called when creating `MountpointError` to get basic information (at least the path) about the error

Reply:
I can add the path in the error for debug purpose if you want

Reply:
We can also enable logger to know in which context we are

Reply:
The user of this crate perform an action that result in an error, It would have partial information about it with the enum (currently only the cause) and to get additional information it would need to call `GetLastError` is it right ?